test/api/kuzzle.test.js

```diff
@@ -1,9 +1,10 @@
 var
   rc = require('rc'),
-  Kuzzle = require.main.require('lib/api/Kuzzle'),
-  should = require('should');
+  rewire = require('rewire'),
+  should = require('should'),
+  Kuzzle = rewire('../../lib/api/kuzzle');
```

Comment:
no need to use `rewire`.
Kuzzle contructor can be loaded by `Kuzzle = require.main.require('lib/api/kuzzle')`